include/bus_handler.hpp

```diff
@@ -65,6 +65,15 @@ class BusHandler
                                      }
                                      return 0;
                                  });
+
+        iface->register_method("ExecuteFunction",
+                               [this](const types::FunctionNumber funcNum) {
+                                   return this->triggerPanelFunc(funcNum);
+                               });
+
+        iface->register_method("getEnabledFunctions", [this]() {
```

Comment:
why this change? and also its corresponding changes ?

Reply:
this is done as part of second commit in the same PR

Reply:
The Story was for completing the first part in this sprint , why the mixup ?

Reply:
will follow properly from next time